mne/io/eeglab/eeglab.py

```diff
@@ -92,8 +109,10 @@ def _get_info(eeg, montage, eog=()):
                 if not np.any(np.isnan(locs)):
                     pos_ch_names.append(chanloc.labels)
                     pos.append(locs)
+        if err_ch_set != set():
+            print('\n')
```

Comment:
(Once I'm certain I understand why I'm getting the unhashable type error, I should be able to avoid it and to generate a test)